### DISCUSSION TITLE
Google login not possible

### DISCUSSION THREAD

**Discussion by Chippel** [arichornlover/uYouEnhanced#811] [General]
Hello all,

I was successfully able to install the app. Unfortunately I can not login via the app because it says Google couldn't confirm it's secure. Any workaround?

**Comment by yodaluca23** [comment]
Assuming your using latest commit go to YT Settings > uYouEnhanced > YouTube Sign in Patch to enabled. Force restart app, login, then turn off the above setting, it causes some other issues, and should only be enabled when signing in.
